packages/dev/core/src/Loading/sceneLoader.ts

```diff
@@ -479,7 +485,8 @@ export class SceneLoader {
     }
 
     private static _FormatErrorMessage(fileInfo: IFileInfo, message?: string, exception?: any): string {
-        let errorMessage = "Unable to load from " + fileInfo.url;
+        const fromLoad = fileInfo.rawData ? " binary data" : fileInfo.url;

```

Comment:
super-nitty - no need for the space:
```suggestion
        const fromLoad = fileInfo.rawData ? "binary data" : fileInfo.url;
```